assets/aml-benchmark/components/src/inference_postprocessor/inference_postprocessor.py

```diff
@@ -368,14 +368,27 @@ def extract_inferences(
     ) -> pd.DataFrame:
         """Extract inferences using generic method if no template or custom post-processor is provided."""
         predicted_data = read_jsonl_files(resolve_io_path(self.prediction_dataset))
+        if len(predicted_data) == 0:
+            mssg = "Received 0 records in the prediction datset."

```

Comment:
nit: `datset` -> `dataset`. May be also add a test for this exception?

Reply:
@iamrk04 @arun-rajora I'm removing this check of len(predicted_data) == 0 as this is being taken care in read_json_files in utils package. It throws error of 'No jsonl file found'.